OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -82,7 +82,12 @@ public void Run(Action<string> emitError, Action<string> emitWarning, ModData mo
 											{
 												if (string.IsNullOrEmpty(imageOverride))
 												{
-													emitWarning("Custom sprite image of actor {0} is null.".F(actorInfo.Value.Name));
+													foreach (var sequenceProvider in sequenceProviders)
+													{
+														var image = renderInfo.GetImage(actorInfo.Value, sequenceProvider, faction);
```

Comment:
Are we sure that *all* places that use `imageOverride` fallback to the actor name in case the image is null? It might be better to introduce a new flag that can be passed to `SequenceReference` if a fallback is in place.

Reply:
I think `rs.GetImage(self)` is too dynamic for static actor info teseting.

Reply:
Can somebody explain what the conclusion was here?

My understanding of the code an interpretation of these comments is that the changes here break the sequence linting in a way that can't easily be fixed...?

Reply:
Fixed it, kinda easily.